Source/WebCore/rendering/RenderLayer.h

```diff
@@ -246,6 +246,8 @@ class RenderLayer : public CanMakeWeakPtr<RenderLayer> {
     // Convert a point in absolute coords into layer coords, taking transforms into account
     LayoutPoint absoluteToContents(const LayoutPoint&) const;
 
+    bool isInsideSVGForeignObject() const { return m_insideSVGForeignObject; }
```

Comment:
I'd prefer to avoid this reference to SVG foreignObject here. It should be more generic, like `needsToRenderOwnScrollbars` or something.

Reply:
This function was already on `RenderLayer` (just made it public) so its probably ok to keep.